sources/bot/modules/spreadsheet/study_staff/study_staff_spreadsheet_handler.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Dict
+
+from bot.modules.spreadsheet.spreadsheet_handler import SpreadsheetHandler
+
+
+class StudyStaffSpreadsheetHandler:
+    def __init__(self, spreadsheet_id: str, file_name: str):
+        self._attributes = {
+            "Студенты": ["username", "ФИО", "Группа", "Подгруппа"],
+            "Преподаватели": ["username", "ФИО"],
+        }
+        self._handler = SpreadsheetHandler(spreadsheet_id, file_name, self._attributes)
+        self._student_sheet_title = list(self._attributes.keys())[0]
+        self._teacher_sheet_title = list(self._attributes.keys())[1]
+
+    def create_spreadsheet(self, spreadsheet_title="Информация о людях", row_count=1000, column_count=10) -> None:
+        self._handler.create_spreadsheet(spreadsheet_title, row_count, column_count)
+
+    def add_student(self, username: str, name: str, group: str, subgroup: str) -> None:
```

Comment:
Why are we passing so much args? To make a comfortable interface, it's better to pass **kwargs or *args. It's so add_teacher

Reply:
Let's speak about it

Reply:
If pass **kwargs or *args you whouldnt see how much and what concrete paramentr need to pass using this class as API

Reply:
They need to be passed in order to make this class easy to replace by others across polymorphism without stopping at these parameters.

Reply:
fixed